packages/ui/src/components/SubToNotification/SubToNotification.tsx

```diff
@@ -13,43 +13,54 @@ export const SubToNotification = () => {
     useState<ServiceWorkerRegistration | null>(null)
 
   useEffect(() => {
-    if (
-      typeof window !== 'undefined' &&
-      'serviceWorker' in navigator &&
-      (window as any).workbox !== undefined
-    ) {
+    const registerServiceWorker = async () => {
+      if (!('serviceWorker' in navigator) || !('workbox' in window)) {
```

Comment:
Double check if it could be `PushManager` instead of `workbox`

```suggestion
      if (!('serviceWorker' in navigator) || !('PushManager' in window)) {
```